src/test/java/org/jenkinsci/plugins/kubernetes/credentials/OpenShiftBearerTokenCredentialMockServer.java

```diff
@@ -0,0 +1,109 @@
+package org.jenkinsci.plugins.kubernetes.credentials;
```

Comment:
This file uses 8 space indentation when the rest of the files in this package seem to use 4 space indentation.  As a personal preference, I really like to keep formatting consistent within a plugin so that I'm not seeing different spacing in different files within the same plugin.  This is entirely optional, but if you'd like a copy of the file that is formatted with 4 space indentation, you can copy it from my fork.

* https://github.com/nevingeorgesunny/kubernetes-credentials-plugin/pull/1

Reply:
done , i have used you file as reference. I will also change my intellij setting